PSReadLine/PublicAPI.cs

```diff
@@ -235,5 +235,34 @@ public static bool TryGetArgAsInt(object arg, out int numericArg, int defaultNum
             numericArg = 0;
             return false;
         }
+
+        /// <summary>
+        /// Accept the suggestion text if there is one.
+        /// </summary>
+        public static void AcceptSuggestion()
+        {
+            if (_singleton._suggestionText != null)
+            {
+                using var _ = _singleton.PredictionOff();
+                Replace(0, _singleton._buffer.Length, _singleton._suggestionText);
+            }
+        }
+
+        /// <summary>
+        /// Accept the current or the next suggestion text word.
+        /// </summary>
+        public static void AcceptNextSuggestionWord(int numericArg)
+        {
```

Comment:
Is this meant to be bindable? If so it has the wrong signature.

Reply:
Same here, will update the code.